base/applications/rapps/lang/ro-RO.rc

```diff
@@ -19,88 +19,88 @@ BEGIN
     BEGIN
-        MENUITEM "&Deschide în navigator", ID_OPEN_LINK
-        MENUITEM "&Copiază în memorie", ID_COPY_LINK
+        MENUITEM "&Deschidere a legăturii în navigator", ID_OPEN_LINK
+        MENUITEM "&Copiere a legăturii în clipboard", ID_COPY_LINK
     END
 END
 
 IDR_APPLICATIONMENU MENU
 BEGIN
     POPUP ""
     BEGIN
-        MENUITEM "&Instalează\tCtrl+Enter", ID_INSTALL
-        MENUITEM "&Dezinstalează\tCtrl+Del", ID_UNINSTALL
-        MENUITEM "&Modifică", ID_MODIFY
+        MENUITEM "&Instalare\tCtrl+Enter", ID_INSTALL
+        MENUITEM "&Dezinstalare\tCtrl+Del", ID_UNINSTALL
+        MENUITEM "&Modificare", ID_MODIFY
         MENUITEM SEPARATOR
-        MENUITEM "&Elimină din registru", ID_REGREMOVE
+        MENUITEM "&Eliminare din registru", ID_REGREMOVE
         MENUITEM SEPARATOR
-        MENUITEM "Împ&rospătează\tF5", ID_REFRESH
+        MENUITEM "Împ&rospătare\tF5", ID_REFRESH
         MENUITEM SEPARATOR
-        MENUITEM "Actualizează &baza de date\tCtrl+F5", ID_RESETDB
+        MENUITEM "&Actualizare a bazei de date\tCtrl+F5", ID_RESETDB
     END
 END
 
 IDD_SETTINGS_DIALOG DIALOGEX 0, 0, 250, 265
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Preferințe"
+CAPTION "Setări"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Generale", -1, 4, 2, 240, 61
-    AUTOCHECKBOX "&Păstrează poziția ferestrei", IDC_SAVE_WINDOW_POS, 15, 12, 219, 12
-    AUTOCHECKBOX "Actualizea&ză la pornire lista programelor disponibile", IDC_UPDATE_AVLIST, 15, 29, 222, 12
-    AUTOCHECKBOX "Menține j&urnal de instalare/dezinstalare a programelor", IDC_LOG_ENABLED, 15, 46, 219, 12
+    GROUPBOX "General", -1, 4, 2, 240, 61
+    AUTOCHECKBOX "&Salvare a poziției ferestrei", IDC_SAVE_WINDOW_POS, 15, 12, 219, 12
+    AUTOCHECKBOX "&Actualizare la pornire a listei programelor disponibile", IDC_UPDATE_AVLIST, 15, 29, 222, 12
+    AUTOCHECKBOX "Menținere a j&urnalului de instalare/dezinstalare a programelor", IDC_LOG_ENABLED, 15, 46, 219, 12
     GROUPBOX "Descărcări", -1, 4, 65, 240, 51
     LTEXT "Folderul pentru descărcări:", -1, 16, 75, 100, 9
     EDITTEXT IDC_DOWNLOAD_DIR_EDIT, 15, 86, 166, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | ES_AUTOHSCROLL
-    PUSHBUTTON "Spe&cificare…", IDC_CHOOSE, 187, 85, 50, 14
-    AUTOCHECKBOX "Șt&erge aplicația de instalare după instalarea programului", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
+    PUSHBUTTON "&Răsfoire", IDC_CHOOSE, 187, 85, 50, 14
+    AUTOCHECKBOX "Se șterge de instalare &după instalarea programului", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
     GROUPBOX "Sursa programului", -1, 4, 118, 240, 46
-    AUTORADIOBUTTON "Folosește valorile implicite", IDC_SOURCE_DEFAULT, 15, 132, 121, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Sursa specificată", IDC_USE_SOURCE, 15, 147, 74, 10
+    CONTROL "Se folosesc valorile implicite", IDC_SOURCE_DEFAULT, 15, 132, 121, 10, WS_GROUP | WS_TABSTOP
+    CONTROL "Sursa specificată", IDC_USE_SOURCE, 15, 147, 74, 10
```

Comment:
Nope! Using `CONTROL` without specifying a control style makes the resource compiler fail to build the resources:
```
/home/runner/work/reactos/reactos/src/base/applications/rapps/lang/ro-RO.rc:80: syntax error
```
Please revert all the specific changes back to `AUTORADIOBUTTON`s.

Reply:
Fixed!